filebeat/prospector/redis/prospector.go

```diff
@@ -0,0 +1,69 @@
+package redis
+
+import (
+	"github.com/elastic/beats/filebeat/channel"
+	"github.com/elastic/beats/filebeat/harvester"
+	"github.com/elastic/beats/filebeat/input/file"
+	"github.com/elastic/beats/libbeat/common"
+	"github.com/elastic/beats/libbeat/logp"
+
+	rd "github.com/garyburd/redigo/redis"
+)
+
+// Stdin is a prospector for stdin
+type Prospector struct {
+	harvester *harvester.Harvester
+	started   bool
+	outlet    channel.Outleter
+	config    Config
+}
+
+// NewStdin creates a new stdin prospector
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
comment on exported function NewProspector should be of the form "NewProspector ..."